aip/general/0126.md

```diff
@@ -55,6 +55,11 @@ message Book {
   - An exception to this rule is if there is a clearly useful zero value. In
     particular, if an enum needs to present an `UNKNOWN`, it is usually clearer
     and more useful for it to be a zero value rather than having both.
+  - The other values **should not** be prefixed by the name of the enum itself.
+    This generally requires users to write `MyState.MYSTATE_ACTIVE` in their
+    code, which is unnecessarily verbose. C++ is an exception. C++ users
```

Comment:
I'm afraid I don't understand this - the document is about naming API enum values, and APIs are inherently "not in a single language". Any API *may* have C++ users and *may* have non-C++ users.

If I were an API designer trying to work out whether or not to include a prefix for enum values, I wouldn't understand how C++ comes into it.

(As an aside, the C# generator automatically drops the prefix if it spots it, when generating a C# enum... so it ends up being nicely readable (`MyState.Active` for example).

Reply:
Doh. Jon is definitely right on this. I had my Googler hat on too tight.